minecraft_servers/blackplasma/manifest.json

```diff
@@ -3,7 +3,9 @@
   "nice_name": "Black Plasma Studios Network",
   "direct_ip": "blackplasma.community",
   "server_wildcards": [
-    "%.blackplasma.community"
+    "%.blackplasma.community",
+    "%.blackplasma.xyz",
+    "blackplasma.beastmc.com"
```

Comment:
Wildcards has to start with %. - If %.blackplasma.beastmc.com is not possible, please remove.